.github/workflows/ci.yml

```diff
@@ -14,27 +14,113 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  library:
+  environment:
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "Preparing Environment Variables..."
+    outputs:
+        XCODE_15_VERSION: 15.4
```

Comment:
A workaround, as environment variables are not usable within a matrix, annoyingly.